modules/common/react/lib/styles/errorRing.ts

```diff
@@ -1,17 +1,22 @@
+import {CanvasTheme} from '@workday/canvas-kit-labs-react-core';
 import {ErrorType} from '../types';
 import {CSSObject} from '@emotion/core';
 import {colors, inputColors} from '@workday/canvas-kit-react-core';
 
-export default function errorRing(error?: ErrorType): CSSObject {
+export default function errorRing(error?: ErrorType, theme?: CanvasTheme): CSSObject {
   let errorBorderColor;
   let errorBoxShadow;
 
   if (error === ErrorType.Error) {
-    errorBorderColor = inputColors.error.border;
-    errorBoxShadow = `inset 0 0 0 1px ${inputColors.error.border}`;
+    errorBorderColor = theme ? theme.palette.error.darkest : inputColors.error.border;
+    errorBoxShadow = `inset 0 0 0 1px ${
+      theme ? theme.palette.error.main : inputColors.error.border
+    }`;
   } else if (error === ErrorType.Alert) {
-    errorBorderColor = colors.cantaloupe600;
-    errorBoxShadow = `inset 0 0 0 2px ${inputColors.warning.border}`;
+    errorBorderColor = theme ? theme.palette.alert.darkest : colors.cantaloupe600;
+    errorBoxShadow = `inset 0 0 0 2px ${
+      theme ? theme.palette.alert.main : inputColors.warning.border
+    }`;
```

Comment:
this bring up an interesting problem. For alert, we have two different colors for accessibility reasons, how do we communicate or handle something like this? Open to suggestions